cmake/common.cmake

```diff
@@ -106,6 +111,8 @@ include_directories(
   ${CCF_DIR}/3rdparty
   ${CCF_DIR}/3rdparty/evercrypt-msr
   ${MSGPACK_INCLUDE_DIR}
+  ${FLATBUFFERS_INCLUDE_DIR}
+  ${CMAKE_CURRENT_BINARY_DIR}
```

Comment:
Why is this required?

Reply:
I am guessing you are referring to `CMAKE_CURRENT_BINARY_DIR`. It is because we are building `data_generated.h` during cmake and it will reside in the projects build directory. It is similar to `ccf_[u|v].h`

Reply:
Oh I see, it's the location of the file generated by `flatc`. Should we keep the output of `flatc` under a specific sub-directory, e.g. `build/flatbuffers`?